controllers/articles_controller.js

```diff
@@ -0,0 +1,11 @@
+const {selectArticleById} = require('../models/articles_model');
+
+exports.getArticleById = function getArticleById(request, response, next) {
```

Comment:
When we export a function, we can also do it by using arrow function like:
```
exports.getArticleById = (request, response, next)=>{
......
}
```
Which will save you some time from typing the name of the function twice

Reply:
Ace, thanks very much for the tip Scarlett!